docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -38,6 +38,13 @@ Run the following from PowerShell to deploy all the prerequisites for building O
 ./scripts/install-windows-prereqs.ps1
 ```
 
+On Windows 10, the Intel PSW is delivered via Windows update. Running the
```

Comment:
Will the right version of PSW be there even on older versions of Windows 10? I don’t know if we need a specific version of PSW in order for the sdk to work correctly

Reply:
As far as I know the PSW has a stable interface and new versions won't break that interface. That said I don't know for sure if it matters.

Reply:
@radhikaj Do you think its ok to merge this?

Unfortunately, we have no control over what PSW version is installed by Windows Update. So fair we have not seen the PSW make breaking changes, but I can't find Intel documentation that says they won't. In theory the SDK should also work with older versions of the PSW, but we only test against the latest.